src/Cms/Page.php

```diff
@@ -826,7 +827,7 @@ public function modified(
 		string|null $handler = null,
 		string|null $languageCode = null
 	): int|string|false|null {
-		$identifier = $this->isDraft() === true ? 'changes' : 'published';
+		$identifier = $this->isDraft() === true ? VersionId::changes() : VersionId::published();
```

Comment:
I think this is the same check covered by `VersionId::default()` and we could replace it with that

Reply:
✅